user_profile.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Profile</title>
+	<link rel="stylesheet" type="text/css" href="usersearch.css">
+</head>
+<body>
+	<!--THIS IS FOR THE USER PROFILE PAGE THEY ENTER THEIR MEMBERSHIP ID IN THE SEARCH FIELD AND IT WILL DISPLAY THEIR PERSONAL INFORMATION ONTO THE NEXT PAGE-->
+	<h1> Type in your Membership ID:</h1>
+	<div class="container">
+  		
+  		<form action="userGet.php" method="GET">
+       <input type="text" name="mid" maxlength="30">
+   
+ 	<div class="search"></div>
+	</div>
+	
```

Comment:
Page displaying well with a search field that allows the user to enter their personal user Id's.